config/default/common/config/metadata/stories/swath_gaps/step003.md

```diff
@@ -1 +1 @@
-<p>When you view imagery from the newer sensor/satellite - the joint NASA/NOAA Suomi National Poplar-orbiting Partnership (Suomi NPP)/Visibie Infrared Imaging Radiometer Suite (VIIRS), there are no black swath gaps because the swath width is 3,040 km wide. This enables an approximate 15% image overlap between consecutive orbits at the equator. This "ribbon" is much wider than the MODIS "ribbon".</p>
+<p>When you view imagery from the newer sensor, the Visibie Infrared Imaging Radiometer Suite (VIIRS), aboard the joint NASA/NOAA Suomi National Polar-orbiting Partnership (Suomi NPP) satellite and the NOAA-20 satellite there are no black swath gaps. The swath width is 3,040 km wide which enables an approximate 15% image overlap between consecutive orbits at the equator. These "ribbons" are much wider than the MODIS "ribbons".</p>
```

Comment:
Visible is mispelled.
"the Visibie Infrared Imaging Radiometer Suite (VIIRS)," should be "the Visible Infrared Imaging Radiometer Suite (VIIRS),"

Reply:
how long has that been there :)